### DISCUSSION TITLE
Hard to find this section with discussions and not clickable support link.

### DISCUSSION THREAD

**Discussion by Constantine-Ketskalo** [community/community#113426] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I had a hard time finding this area with discussions to report a bug. When I was googling something like "github copilot report bug", search results where getting me to "issues" tab of github copilot page about tracking my issues instead of reporting github copilot issues. Moreover, at the [github copilot page](https://github.com/features/issues) if you scroll down, "Support" is just a text, it's not clickable. Looks confusing, even like some trolling in a situation when I couldn't find a way to report a bug.

**Comment by aldrshan** [comment]
Type discussion or community in the search bar it should be in the top results.

**Reply by Constantine-Ketskalo** [reply]
Sure. I understand. And on one hand it makes sence because it makes terms "discussion" and "community" to stand out and eisear to find and move all the requests about bugs to actual functionality of github copilot. But it works well for those who know about it already. While if I'm new to this and I don't know about those 2 words yet, searching for this section turns into hell.
Besides words "discusson" and "community" sounds more like a place to simply chat about something, discuss it, but not a bug reporting place.
So I'd suggest at least to add to the github copilot page/pages meta keywords for phrases like "github copilot report bug", so that it gets users to the right place immediately.
Personally I managed to find this community-discussions sections only by posting a question at stackoverflow, which doesn't seem to be the most correct, convenient and user-friendly way of finding things.
